packages/donate-button/src/components/Donations/Logo/logo.css

```diff
@@ -1,38 +1,41 @@
 .logo {
-    display: flex;
-    align-items: flex-start;
+  display: flex;
+  align-items: flex-start;
 
-    /* Fix logo to the top and scroll the content beneath*/
-    position: sticky;
-    top: 0;
-    background: white;
+  /* Fix logo to the top and scroll the content beneath*/
+  position: sticky;
+  top: 0;
+  background: white;
 
-    margin: -1rem -2rem 0 -2rem;
-    padding: 1.5rem 2rem;
+  margin: -1rem -2rem 0 -2rem;
+  padding: 1.5rem 2rem;
+
+  /* Fix border radius on safari */
+  border-top-left-radius: 1.5rem;
```

Comment:
seems suspicious that the border radius wouldn't work in all cases, but I'll let this slide since we're moving to a more reliable system in the `donate-button-beta` folder.